recipes-scanners/clamav/clamav_1.4.1.bb

```diff
@@ -0,0 +1,226 @@
+SUMMARY = "ClamAV anti-virus utility for Unix - command-line interface"
+DESCRIPTION = "ClamAV is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats."
+HOMEPAGE = "http://www.clamav.net/index.html"
+SECTION = "security"
+LICENSE = "LGPL-2.1-only"
+
+DEPENDS = "glibc llvm libtool db openssl zlib curl libxml2 bison pcre2 json-c libcheck rust-native cargo-native libmspack"
+ 
+COMPATIBLE_HOST:libc-musl:class-target = "null"
+
+LIC_FILES_CHKSUM = "file://COPYING.txt;beginline=2;endline=3;md5=f7029fbbc5898b273d5902896f7bbe17"
+
+# July 30th, 2022
+SRCREV = "e81cc6f4271fbdb2e40a9734e06bb507441d6e09"
+
+SRC_URI = "git://github.com/Cisco-Talos/clamav;branch=rel/1.4;protocol=https \
+    file://clamd.conf \
+    file://freshclam.conf \
+    file://volatiles.03_clamav \
+    file://tmpfiles.clamav \
+    file://headers_fixup.patch \
+    file://oe_cmake_fixup_1.4.1.patch \
+"
+S = "${WORKDIR}/git"
+
+LEAD_SONAME = "libclamav.so"
+SO_VER = "12.0.0"
+
+BINCONFIG = "${bindir}/clamav-config"
+
+inherit cmake chrpath pkgconfig useradd systemd multilib_header multilib_script
+
+# =============================================================================
+# Rust Cross-Compilation Configuration for Yocto
+# =============================================================================
+# 
+# ClamAV 1.4.1 introduces Rust components (libclamav_rust) that require special
+# handling in Yocto cross-compilation environments. This addresses the classic
+# "host vs target triple confusion" issue in Yocto+Rust builds.
+#
+# PROBLEM:
+# - Build host: x86_64-linux (Ubuntu 24.04)  
+# - Build target: x86_64-nilrt-linux (NILRT embedded)
+# - Rust sees both as: x86_64-unknown-linux-gnu (same triple!)
+#
+# This causes Rust build scripts (procedural macros, build.rs) to incorrectly
+# link against target sysroot libraries instead of host libraries, resulting in:
+# "cannot find Scrt1.o", "cannot find -lc", etc.
+#
+# SOLUTION:
+# Configure Rust to use native host toolchain (gcc) for build scripts while
+# allowing ClamAV's CMake to handle target cross-compilation separately.
+#
+# REFERENCES:
+# - https://github.com/rust-lang/cargo/issues/3349 (Cross compilation issues)  
+# - https://github.com/rust-lang/cargo/issues/4423 (Host vs target confusion)
+# - Yocto Project Rust Cross-Compilation Documentation
+#
+# MAINTAINER NOTE:
+# This configuration is specific to x86_64-to-x86_64 cross-compilation in Yocto.
+# For different host/target combinations, the cargo config may need adjustment.
+# =============================================================================
```

Comment:
I think you can drop this note. It's a little too verbose and mentions 'NILRT' - which others in the yocto community won't know about. If it relates to a specific recipe setting that you're making elsewhere, you can summarize it down to a one-liner comment.